src/components/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { AddPostMobile } from "./AddPostMobile";
+import { AddQuestionModal } from "./AddQuestionModal";
+import { CustomSnackbar } from "./CustomSnackbar";
+import { DiscussionCard } from "./DiscussionCard";
+import { DiscussionCardsSection } from "./DiscussionCardsSection";
+import { FullscreenLoader } from "./FullscreenLoader";
+import { Navbar } from "./Navbar";
+import { RequireAuth } from "./RequireAuth";
+import { Sidenav } from "./Sidenav";
+import { TrendingSection } from "./TrendingSection";
```

Comment:
```ts
export { AddPostMobile } from "./AddPostMobile";
export { AddQuestionModal } from "./AddQuestionModal";
export { CustomSnackbar } from "./CustomSnackbar";
export { DiscussionCard } from "./DiscussionCard";
.....
```
you can write these import/exports like too

Reply:
Done